changes/3085.feature.rst

```diff
@@ -0,0 +1,2 @@
+Extending location services with the more battery balanced modes like significant location change and reporting of
+visits.
```

Comment:
There's no need to break change notes at 80 chars; we prefer to keep them on one line to make the post-processing by towncrier easier.